test/test_mpi.c

```diff
@@ -35,12 +35,16 @@ static int oob_allgather(void *sbuf, void *rbuf, size_t msglen,
 int xccl_mpi_test_init(int argc, char **argv) {
     char *var;
     int rank, size;
+    xccl_config_t *test_conf;
 
     MPI_Init(&argc, &argv);
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &size);
 
+
     /* Init xccl library */
+    xccl_config_read(NULL, NULL, &test_conf);
```

Comment:
Should this be xccl_config_read ("TEST", NULL, &test_conf) and then the following getenv can be removed?

Reply:
Will be a part of another PR together with new param, config structures